docs/server/installation.rst

```diff
@@ -388,55 +232,23 @@ software. To run it, just issue:
 
    (env) $ pootle start
 
-
```

Comment:
Keep this blank line. We put two blank lines after blocks to increase readability for docs writers.